lib/ts/unit-tests/test_BufferWriterFormat.cc

```diff
@@ -204,6 +204,25 @@ TEST_CASE("BWFormat", "[bwprint][bwformat]")
   bw.print("|{:<0.2,5x}|", sv);
   REQUIRE(bw.view() == "|63313|");
 
+  bw.reduce(0);
+  bw.print("|{}|", true);
+  REQUIRE(bw.view() == "|1|");
+  bw.reduce(0);
+  bw.print("|{}|", false);
+  REQUIRE(bw.view() == "|0|");
+  bw.reduce(0);
+  bw.print("|{:s}|", true);
+  REQUIRE(bw.view() == "|true|");
+  bw.reduce(0);
+  bw.print("|{:S}|", false);
+  REQUIRE(bw.view() == "|FALSE|");
+  bw.reduce(0);
+  bw.print("|{:>9s}|", false);
+  REQUIRE(bw.view() == "|    false|");
+  bw.reduce(0);
+  bw.print("|{:=10s}|", true);
```

Comment:
I think we should use `^` for center alignment, and `=` for fill-after-sign, like the [python format mini-language](https://docs.python.org/3.5/library/string.html#format-specification-mini-language) does. Probably this should be considered beyond the scope of this PR, but we should be consistent with the inspiration for the formatting.

Reply:
Yes, I will get that fixed.

Reply:
This is addressed in #3463.